assets/templates/header.html

```diff
@@ -69,10 +69,11 @@
           <ul class="nav-right-main--list">
             <li>
             <a href="#" class="user-menu--open active" data-count="1"><span class="ms-icon icon-notification"></span></a>
-            <div class="user-menu helper--right">
+            <div class="user-menu helper--right notifications">
               <h5>NUEVO: Comportamiento en Sitio</h5>
```

Comment:
It is missing changing this structure, right?

Reply:
Yes, just change it with the message structure that brings backend